staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/defaults.go

```diff
@@ -71,4 +71,18 @@ func SetDefaults_CustomResourceDefinitionSpec(obj *CustomResourceDefinitionSpec)
 			{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"], JSONPath: ".metadata.creationTimestamp"},
 		}
 	}
+	for i, v := range obj.Versions {
```

Comment:
@sttts The defaulting is not feature gated. As mentioned in https://github.com/kubernetes/kubernetes/pull/67553#discussion_r213480081, in version X where `apiextensionsfeatures.CustomResourceWebhookConversion` is disabled we should still apply defaulting so that CRD doesn't loss per-version Schema/Subresources/AdditionalPrinterColumns when rolling forward to version Y